src/windows/com.rs

```diff
@@ -122,6 +125,7 @@ impl COMPort {
                     handle: cloned_handle,
                     port_name: self.port_name.clone(),
                     timeout: self.timeout,
+                    dtr: self.dtr,
```

Comment:
This clones the `dtr` state when a `COMPort` is cloned, which might not always be the right behaviour.

Another option could be to remove the `dtr` field from this struct, and instead call `DeviceIoControl` with `IOCTL_SERIAL_GET_DTRRTS` in `write_request_to_send` to find the current state of the DTR pin, as described in [this Stack Overflow answer](https://stackoverflow.com/a/59517202). It's another potentially fallible syscall though, so I just went with this approach for now. Happy to make this change if desired however